include/macros.h

```diff
@@ -156,6 +161,48 @@ extern struct GraphicsContext* __gfxCtx;
     }                                                   \
     (void)0
 
+#define GRAPH_ALLOC(gfxCtx, size) Graph_Alloc(gfxCtx, size)
+#define MTXF_TO_MTX(gfxCtx, line, file) Matrix_ToMtxDebug(gfxCtx, line, file)
+#define MATRIX_NEW(gfxCtx, line, file) Matrix_NewMtxDebug(gfxCtx, line, file)
+#define MATRIX_CHECK_FLOATS(mtx, file, line) Matrix_CheckFloats(mtx, file, line)
+#define DMA_REQUEST_SYNC(ram, vrom, size, file, line) DmaMgr_RequestSyncDebug(ram, vrom, size, file, line)
+#define DMA_REQUEST_ASYNC(req, ram, vrom, size, unk5, queue, msg, file, line) DmaMgr_RequestAsyncDebug(req, ram, vrom, size, unk5, queue, msg, file, line)
+#define GAME_STATE_ALLOC(gameState, size, file, line) GameState_Alloc(gameState, size, file, line)
+#define SYSTEM_ARENA_MALLOC(size, file, line) SystemArena_MallocDebug(size, file, line)
+#define SYSTEM_ARENA_MALLOCR(size, file, line) SystemArena_MallocRDebug(size, file, line)
+#define SYSTEM_ARENA_FREE(size, file, line) SystemArena_FreeDebug(size, file, line)
+#define ZELDA_ARENA_MALLOC(size, file, line) ZeldaArena_MallocDebug(size, file, line)
+#define ZELDA_ARENA_MALLOCR(size, file, line) ZeldaArena_MallocRDebug(size, file, line)
```

Comment:
Thinking for both of these it should be `MALLOC_R`, so that the "R" for reverse stands out more. 
It stands out in the function name because of the case difference, but doesnt really do so well in all caps